distributed/scheduler.py

```diff
@@ -2794,13 +2793,11 @@ def rebalance(self, comm=None, keys=None, workers=None):
                 },
             )
 
-            if not all(r["status"] == "OK" for r in result.values()):
-                raise Return(
-                    {
-                        "status": "missing-data",
-                        "keys": sum([r["keys"] for r in result if "keys" in r], []),
```

Comment:
Was this wrong before? I would have expected this to iterate over result.values

Reply:
Yeah, this was wrong before.

Reply:
It's really good to have a second pair of eyes here